git_machete/git_operations.py

```diff
@@ -295,6 +295,10 @@ def get_worktree_git_subpath(self, *fragments: str) -> str:
     def get_main_git_subpath(self, *fragments: str) -> str:
         return os.path.join(self.__get_main_git_dir(), *fragments)
 
+    def get_git_machete_definition_file_path(self) -> str:
+        use_top_level_machete_file = self.get_boolean_config_attr(key='machete.worktree.useTopLevelMacheteFile')
```

Comment:
Looks good, just docs + tests to be added to his draft

Reply:
Btw... while you're here, I'd add a **required** param to `get_boolean_config_attr`: `default_value`, so that whoever read the code, doesn't scratch their heads whether is `False` or `True` that's assumed in the absence of the config key

Reply:
As we previously decided, the docs will be located in the `config` section being implemented in #539 but `file` section needed updating and the result is shown below:
<img width="1143" alt="image" src="https://user-images.githubusercontent.com/86779115/171906435-b2c9fe5b-3795-46fa-84e0-f8949e3a89c6.png">